package/Dockerfile

```diff
@@ -1,29 +1,47 @@
-FROM registry.suse.com/bci/bci-base:15.5
+# Temporary build stage
+FROM registry.suse.com/bci/bci-base:15.5 as builder
 
-ARG kube_bench_version=0.6.12
+# Define build arguments
+ARG kube_bench_version=0.6.17
 ARG sonobuoy_version=0.56.16
 ARG kubectl_version=1.28.0
```

Comment:
Please follow-up with a PR to `rancher/renovate-config` so that this is automatically bumped going forwards.